docs/source/installation.rst

```diff
@@ -13,26 +13,56 @@ First, install the buildozer project with::
 Targeting Android
 -----------------
 
-If you target Android, you must install at least Cython, few build libs, and a
-Java SDK. Some binaries of the Android SDK are still in 32 bits, so you need
-few 32bits libraries available:
+Android on Ubuntu (64bit)
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Android on Ubuntu 18.04 (64bit)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(tested on 18.04, 18.10, 19.04)
```

Comment:
I would rather just keep the LTS title `18.04`, because it's the one that will truly be under CI not the other.
I mean maybe you or somebody tested 18.10, 19.04 but not on regular basis like a CI would do.
So the idea is not that we don't support other version, but more to say that we're confident it works for a given one

Reply:
right, it's true that it's not going to be tested continuously, so it might break in the future., i'll edit.